include/powsybl/stdcxx/reference_wrapper.hpp

```diff
@@ -16,37 +16,37 @@
 namespace stdcxx {
 
 template <typename T>
-class reference_wrapper {
+class Reference {
```

Comment:
I think you have to rename this file to `reference.hpp` even if it will introduce a lot of changes

Reply:
Done